docs/CONTRIBUTING.md

```diff
@@ -141,6 +141,13 @@ Example:
     GOOD: * [A Way-backed Interesting Book](https://web.archive.org/web/20211016123456/http://example.com/) - John Doe (HTML) *(:card_file_box: archived)*
     ```
 
+### Alphabetical order
+
+- When there are multiple titles beginning with the same letter order them by the second, and so on.
+- `one two` comes before `onetwo`
```

Comment:
```suggestion
For example:
- `aa` comes before `ab`
```

Reply:
this example makes more sense to put it in the line above (146), to reinforce the concept.
`one two` describes another problem, that of spaces